coalib/bears/requirements/PythonRequirement.py

```diff
@@ -83,6 +103,15 @@ def multiple(cls, *args):
                     reqs.append(cls(name, version),)
                 except ValueError:
                     raise TypeError('The tuple must have 2 elements.')
+            elif isinstance(requirement, list):
+                if len(requirement) == 1:
+                    reqs.append(cls(requirement),)
+                else:
+                    try:
+                        name, version = requirement
+                        reqs.append(cls(name, version),)
+                    except ValueError:
+                        raise TypeError('The list must have max 2 elements.')
```

Comment:
I thought of it too, but I'm a bit struggling that it's even possible to unpack sets, which yields in unknown unpacking behaviour, because items inside sets are not positional fixed. So imo it makes sense to restrict types here to tuples and lists.
But I agree, code duplication is bad, just do `if isinstance(..., tuple) or isinstance(..., list)` and change the TypeError message to something more general